study.md

```diff
@@ -57,7 +60,7 @@ Which ActiveRecord method creates new objects? Does this method persist objects
 as rows in the database, or is there another method required for persistence?
 
 ```md
-<!-- your response here -->
+create. Creates new objects and if you need to create additonal methods you can use .new method.
```

Comment:
The latter part of the sentence doesn't make sense.  `#new` instantiates a new object but doesn't save it to the database.  `#save` needs to be invoked on it afterward to save it.